### DISCUSSION TITLE
Docker on older Windows versions

### DISCUSSION THREAD

**Discussion by nisha0119** [opensafely/documentation#375] [Installing OpenSAFELY] [answered]
Hi all,

Probably the last hoop to jump before getting started on my project. Been blocked from using both locally downloaded software and Gitpod on organisational laptop. So resorted to getting personal laptop fixed and functioning. However, it's fairly old (Windows 8 Pro) and posing its own issues with Docker (image error below).

Now at a crossroad as to a) whether I still need to pursue Gitpod (just via my personal laptop where I think it will work) or b) if there's a workaround to getting Docker installed locally? 
I've checked out the forum and detailed Docker guide, though my query may be more specific.

Any help, much appreciated! Thanks 
Nisha 

![Docker blocker](https://user-images.githubusercontent.com/83028350/130457958-e2ac9d91-49b2-493c-86b1-08d7ea57b983.jpg)

fyi @acagreen17

**Comment by acagreen17** [comment]
So I think you need to install docker Toolbox rather than Desktop (but happy for someone with more docker experience to correct me). [This](https://devconnected.com/how-to-install-docker-on-windows-7-8-10-home-and-pro/#1_Install_Docker_on_Windows_7810_Home) guide looks like it might be useful...

**Reply by nisha0119** [reply]
Thanks Millie. Yes, it may be the least preferred option based on the guide but if it offers most of the functionalities that desktop does (and works in terms of installation) I may go down that route

**Comment by acagreen17** [comment, marked as answer]
Morning Nisha, I've spoken to the team and it would be best to pursue getting gitpod to work. Let me know if a call would be useful to see where we can help with this?

**Comment by StevenMaude** [comment]
Just to clarify for any future readers that Docker Toolbox [stopped being supported by Docker](https://github.com/docker/toolbox/commit/c592027e6e3e9777497c1ef7d27bec56bafdd989) in 2020.

At time of writing, the current option if you want to run OpenSAFELY directly on your own Windows laptop — not via Gitpod in browser — would be to install Docker Desktop. [Docker Desktop on Windows](https://docs.docker.com/desktop/windows/install/) requires Windows 10.
